pkg/rancher-desktop/assets/scripts/moproxy.initd

```diff
@@ -0,0 +1,75 @@
+#!/sbin/openrc-run
+
+name=moproxy
+
+description="A transparent TCP to SOCKSv5/HTTP proxy."
+
+extra_started_commands="enable disable reload"
+description_enable="Start redirecting the network traffic to the HTTP proxy."
+description_disable="Stop redirecting the network traffic to the HTTP proxy."
+description_reload="Reload the proxy list."
+
+# TCP Listen address
+: ${host:=${MOPROXY_HOST:-"::"}}
+# TCP Listen port
+: ${port:=${MOPROXY_PORT:-"2080"}}
+# List of backend proxy servers
+: ${proxy_list:=${MOPROXY_PROXYLIST:-"/etc/moproxy/proxy.ini"}}
+# Additional arguments to pass to moproxy
+: ${moproxy_args:=${MOPROXY_ARGS:-""}}
+# coma seperated list of port traffic to redirect to moproxy
+: ${ports_redirected:=${MOPROXY_REDIRECTED_PORT:-"80,443"}}
+
+command="'${MOPROXY_PEER_BINARY:-/usr/sbin/moproxy}'"
+command_args="--host ${host} --port ${port} --list ${proxy_list} ${moproxy_args}"
+command_background="yes"
+pidfile="/run/${name}.pid"
+
+MOPROXY_PEER_LOGFILE="${MOPROXY_PEER_LOGFILE:-${LOG_DIR:-/var/log}/${RC_SVCNAME}-peer.log}"
+output_log="'${MOPROXY_PEER_LOGFILE}'"
+error_log="'${MOPROXY_PEER_LOGFILE}'"
+
+iptables_cmd() {
+	iptables -t nat -$1 $2 -p tcp -m multiport --dports "${ports_redirected}" -j REDIRECT --to-port "${port}"
```

Comment:
I noticed that you are using a mixture of short-hand and long args. What do you think if we change all the args to the long format? 
That way we can easily see what is going on here and we don't need to go to the terminal to do `iptables --help` every time we are looking at this file.
e.g
```
iptables --table nat --$1 $2 --protocol tcp --match multiport --dports "${ports_redirected}" --jump REDIRECT --to-port "${port}"
```